fixit.py

```diff
@@ -130,13 +145,20 @@ def getHist(self):
     outtext=[]
     resp=requests.post(base_url,json=self.pvList)
     resp.raise_for_status()
-    for pv in self.pvList:
+    for nn,pv in enumerate(self.pvList):
       try:
         val = round(resp.json()[pv]['val'], 4)
         self.histVals.append(val)
       except:
         self.histVals.append(np.nan)
-      outtext.append(f"{pv} was {self.histVals[-1]}")
+      if showChanged:
+        changed=abs(self.histVals[nn]-self.currVals[nn])>self.diffTol
+      if showDeltas:
+        delta=self.currVals[nn]-self.histVals[nn]
+      if (not showChanged or (showChanged & changed)) and showDeltas:     
+        outtext.append(f"{pv} was {self.histVals[-1]} now-then= {delta}")
+      elif (not showChanged or (showChanged and changed)) and not showDeltas:
```

Comment:
Is there a reason for using `&` above and `and` here?

Reply:
Matlab leaking in.  Thanks for the catch!